README.md

```diff
@@ -102,7 +102,7 @@ path "gcppca/issue-with-genkey/my_tls_cert_rsa_1" {
       "validity"= ["P30D"]
       "dns_san" = ["client.domain.com,client2.domain.com"]        
       "subject" = ["C=US,ST=California,L=Mountain View,O=Google LLC,CN=google.com"]
-      "reusable_config" = ["leaf-server-tls"]     
+      "issuing_certificate_authority" = ["prod-root"]
```

Comment:
Can we remove this from the default example? We'd like to encourage customers to not target specific CAs except when testing out a staged CA.